XRTK-Core/Packages/com.xrtk.core/Providers/Controllers/Hands/HandBounds.cs

```diff
@@ -0,0 +1,129 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using UnityEngine;
+using XRTK.Definitions.Utilities;
+using XRTK.EventDatum.Input;
+using XRTK.Interfaces.InputSystem;
+using XRTK.Interfaces.InputSystem.Handlers;
+using XRTK.Interfaces.Providers.Controllers;
+using XRTK.Services;
+
+namespace XRTK.Providers.Controllers.Hands
+{
+    /// <summary>
+    /// Utility behavior to access the axis aligned bounds of IMixedRealityHands.
+    /// </summary>
+    public class HandBounds : MonoBehaviour, IMixedRealitySourceStateHandler, IMixedRealityHandJointHandler
+    {
+        /// <summary>
+        /// Accessor for the bounds associated with a handedness.
+        /// </summary>
+        public Dictionary<Handedness, Bounds> Bounds { get; private set; } = new Dictionary<Handedness, Bounds>();
```

Comment:
public dictionaries should always be `IReadonlyDictionary` as they should be immutable when passed outside the class.

create a private readonly dictionary for the local accessor and the `IReadonlyDictionary` as a property.